doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2939,6 +2939,31 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       make some proof steps explicit when refactoring a proof script
       to make it readable.
 
+   .. tacv:: convert_concl_no_check @term
+
+      For advanced usage. Similar to :n:`change @term`, but as an optimization,
+      it skips checking that :n:`@term` is convertible to the goal.
+
+      Recall the Coq kernel typechecks proofs again when they are concluded to
+      ensure safety. Hence, using :n:`change` checks convertibility twice
+      overall, while :n:`convert_concl_no_check` can produce ill-typed terms,
```

Comment:
You should use `:tacn:` here as well, but you need first to add a `:name:` option to the `.. tacv::` directive, otherwise it won't be indexed (because variants aren't by default).